archeryutils/classifications/agb_indoor_classifications.py

```diff
@@ -92,17 +98,24 @@ def _make_agb_indoor_classification_dict() -> Dict[str, Dict[str, Any]]:
                     bowstyle["genderStep_in"],
                 )
 
-                classification_dict[groupname]["class_HC"] = np.empty(
-                    len(agb_classes_in)
-                )
-                for i in range(len(agb_classes_in)):
+                classifications_count = len(agb_classes_in)
+
+                class_hc = np.empty(classifications_count)
+                for i in range(classifications_count):
                     # Assign handicap for this classification
-                    classification_dict[groupname]["class_HC"][i] = (
+                    class_hc[i] = (
                         bowstyle["datum_in"]
                         + delta_hc_age_gender
                         + (i - 1) * bowstyle["classStep_in"]
                     )
 
+                groupdata: GroupData = {
+                    "classes": agb_classes_in,
+                    "classes_long": agb_classes_in_long,
+                    "class_HC": class_hc,
+                }
+                classification_dict[groupname] = groupdata
```

Comment:
Could this be made one statement to remove temporary variable `groupdata`, thereby resolving the `too-many-locals` issue?

Reply:
I initially tried that but I think to include the type hint for the TypedDict it has to be assigning to a variable, you can't type hint the assignment to a dictionary value.
Possibly it could be done with a cast but that doesn't feel any better.